torchci/rockset/pytorch_dev_infra_kpis/__sql/number_of_force_pushes_historical.sql

```diff
@@ -0,0 +1,16 @@
+SELECT
+    FORMAT_TIMESTAMP('%m-%d-%y', DATE_TRUNC(:granularity, issue_comment._event_time)) AS bucket,
+    COUNT(DISTINCT issue_comment.issue_url) AS count
```

Comment:
Yeah, that's true.  I opt for this easy query because force merge + revert + force merge is rarer, and I cover that in a separate query [commons.reverted_force_push](https://console.rockset.com/lambdas/details/commons.reverted_force_push), i.e. counting the number of force merges leading to reverts.  The common case I see if people trying force merge several times to get the correct usage, i.e. https://github.com/pytorch/pytorch/pull/82407. We have fixed the confusing `-f FORCE` description now, so it's less frequent. However, people still miss the message from time to time running only `merge -f`.